redis_cloud/manifest.json

```diff
@@ -0,0 +1,77 @@
+{
+  "manifest_version": "2.0.0",
+  "app_uuid": "0b59b80e-db72-44a6-8c2b-67475d10ad71",
+  "app_id": "redis-cloud",
+  "display_on_public_website": true,
+  "tile": {
+    "overview": "README.md#Overview",
+    "configuration": "README.md#Setup",
+    "support": "README.md#Support",
+    "changelog": "CHANGELOG.md",
+    "description": "Redis Cloud Integration",
+    "title": "Redis Cloud",
+    "media": [
+      {
+        "media_type":  "image",
+        "caption": "closter overview displays",
+        "image_url": "images/datadog-overview-cluster.png"
+      },
+      {
+        "media_type":  "image",
+        "caption": "graph of node memory usage",
+        "image_url": "images/datadog-detail-node-memory.png"
+      },
+      {
+        "media_type":  "image",
+        "caption": "graphs of database statistics",
+        "image_url": "images/datadog-database-overview.png"
+      }
+    ],
+    "classifier_tags": [
+      "Supported OS::Linux",
+      "Supported OS::Windows",
+      "Supported OS::macOS",
+      "Category::AI/ML",
+      "Category::Caching",
+      "Category::Data Stores",
+      "Category::Cloud",
+      "Offering::Integration",
+      "Queried Data Type::Metrics",
```

Comment:
To confirm are you pulling and submitting metrics?

Reply:
Yes. This is the same set of classifier tags as Enterprise except that it includes cloud.

Reply:
Okay thank you! Can you update the README to let users know that you pull metrics out and why? We want to make sure it's clear for users :)

Reply:
"...pull metrics out and why?". I'm unclear on the significance of your ask. Both have a section that says the following:
Data Collected
The current release gathers all metrics for databases, nodes, and shard.

Reply:
@emarsha94 can you tell me if that's enough?

Reply:
To clarify, it should specify if the integration is pulling metrics out of customer Datadog accounts, and sending metrics into Datadog. Ex: This integration pulls x metrics from Datadog and sends in x metrics to Datadog

Reply:
It does not pull metrics from Datadog, it only pulls metrics from Redis' metrics exporter. Is there some place in the manifest where this needs to be specified?

Reply:
Okay got it. For context, Queried Data Type means that the integration is pulling metrics from Datadog. We can go ahead and remove "Queried Data Type::Metrics", since the integration doesn't pull metrics from Datadog.